### ISSUE TITLE
Schedule a job into the future

### ISSUE THREAD

**Issue by TomKaltz** [grantcarthew/node-rethinkdb-job-queue#1]
Can you add a job that is set to run not immediately but at a pre-defined time in the future?  Similar to how future retries work?

**Comment by grantcarthew** [comment]
Hi @TomKaltz, I'm away from home with little signal and on my phone. Will look at this in about five days. It is a good idea though. My first thought is to set 'dateRetry' on the new Job object prior to adding it to the queue. This will not work as is though.

**Comment by grantcarthew** [comment]
This was easier than I thought @TomKaltz. If you had set the `dateRetry` value on the v0.0.7 package it would have worked. However after your request I decided to rename the `dateRetry` property to `dateEnable`. I added a test for delayed jobs and the package has been updated to v0.0.8.

I have [written a document to explain how to use](https://github.com/grantcarthew/node-rethinkdb-job-queue/wiki/Delayed-Job) the `dateEnable` property to delay the processing of jobs.
